2013/09/24 08:45:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'cave.nl'' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CModule.php(388): CDbConnection->init()
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(CWizardSession))
#7 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardSession.php(23): CActiveRecord::model('CWizardSession')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(79): CWizardSession::model()
#9 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(19): CWizardController->getSession()
#10 C:\xampp\htdocs\htdocs_old\protected\controllers\TestController.php(10): CWizardController->init()
#11 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(275): TestController->init()
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test')
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#15 {main}
REQUEST_URI=/htdocs_old/test/
---
2013/09/24 08:45:28 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.contact.ContactModule" is invalid. Make sure it points to an existing PHP file.' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\YiiBase.php(198): YiiBase::import('application.mod...', true)
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CModule.php(283): YiiBase::createComponent('application.mod...', 'contact', NULL, Array)
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(331): CModule->getModule('contact')
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(270): CWebApplication->createController('contact/login')
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#7 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:46:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\pr
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:46:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lotected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')ib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:47:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:47:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:48:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:48:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:48:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:48:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact/login".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('contact/login')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#3 {main}
REQUEST_URI=/htdocs_old/contact/login
---
2013/09/24 08:54:37 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 08:54:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(119): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(58): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(69): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 08:55:52 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 08:55:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(119): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(58): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(69): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:00:05 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:00:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:00:09 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:00:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:03:21 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 27 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:03:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 27 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:03:23 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 28 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:03:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 28 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:03:26 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 29 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:03:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 29 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:03:42 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 30 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:03:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 30 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:03:45 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 31 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:03:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 31 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(55): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#18 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:04:41 [error] [php] include(VacancyForm.php): failed to open stream: No such file or directory (C:\xampp\htdocs\htdocs_old\protected\lib\yii\YiiBase.php:418)
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php(40): spl_autoload_call()
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php(112): CWizardModelAction->getInstance()
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php(58): CWizardModelAction->__get()
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php(112): CWizardModelAction->getData()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(62): CWizardModelAction->__get()
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): TestController->runAction()
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): TestController->runActionWithFilters()
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): TestController->run()
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): TestController->forward()
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): TestController->actionIndex()
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams()
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): TestController->runAction()
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): TestController->runAction()
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): TestController->runActionWithFilters()
#14 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): TestController->run()
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController()
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\htdocs_old\index.php(48): CWebApplication->run()
REQUEST_URI=/htdocs_old/test/index
2013/09/24 09:13:12 [error] [exception.CException] exception 'CException' with message 'Property "RankingForm.transactionMethod" is not defined.' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\views\test\_ranking.php(11): CComponent->__get('transactionMeth...')
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(870): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\htdocs_old\protected\components\CommonController.php(29): CController->renderPartial('_ranking', Array, true, false)
#5 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(208): CommonController->renderPartial('_ranking', Array, true, false)
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(783): CWizardController->renderPartial('_ranking', Array, true)
#7 C:\xampp\htdocs\htdocs_old\protected\components\CommonController.php(38): CController->render('_ranking', Array, false)
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(203): CommonController->render('_ranking', Array, false)
#9 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php(72): CWizardController->render('_ranking', Array)
#10 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(63): CWizardModelAction->run()
#11 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#14 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#21 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#22 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#23 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#24 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:13:25 [error] [exception.CException] exception 'CException' with message 'TestController contains improperly nested widget tags in its view "C:\xampp\htdocs\htdocs_old\protected\views\test\_ranking.php". A TbActiveForm widget does not have an endWidget() call.' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CBaseController.php:102
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(870): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#1 C:\xampp\htdocs\htdocs_old\protected\components\CommonController.php(29): CController->renderPartial('_ranking', Array, true, false)
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(208): CommonController->renderPartial('_ranking', Array, true, false)
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(783): CWizardController->renderPartial('_ranking', Array, true)
#4 C:\xampp\htdocs\htdocs_old\protected\components\CommonController.php(38): CController->render('_ranking', Array, false)
#5 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(203): CommonController->render('_ranking', Array, false)
#6 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php(72): CWizardController->render('_ranking', Array)
#7 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(63): CWizardModelAction->run()
#8 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#10 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(748): CController->run('ranking')
#11 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(29): CController->forward('ranking')
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\actions\CInlineAction.php(50): CWizardController->actionIndex()
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(65): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('index')
#18 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/index')
#19 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#20 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#21 {main}
REQUEST_URI=/htdocs_old/test/index
---
2013/09/24 09:13:52 [error] [php] Undefined index: RankingForm (C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php:50)
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): TestController->runActionWithFilters()
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): TestController->run()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController()
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\xampp\htdocs\htdocs_old\index.php(48): CWebApplication->run()
REQUEST_URI=/htdocs_old/test/ranking?filters=1
2013/09/24 09:15:14 [error] [php] Undefined index: RankingForm (C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\actions\CWizardModelAction.php:50)
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): TestController->runActionWithFilters()
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): TestController->run()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController()
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\xampp\htdocs\htdocs_old\index.php(48): CWebApplication->run()
REQUEST_URI=/htdocs_old/test/ranking?filters=1
2013/09/24 09:15:23 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/index
---
2013/09/24 09:17:56 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/index
---
2013/09/24 09:18:42 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/index
---
2013/09/24 09:19:08 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/
---
2013/09/24 09:21:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'keysjber_xii2'@'145.37.93.31' (using password: YES)' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CModule.php(388): CDbConnection->init()
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(CWizardSession))
#7 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardSession.php(23): CActiveRecord::model('CWizardSession')
#8 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(75): CWizardSession::model()
#9 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(20): CWizardController->getSession()
#10 C:\xampp\htdocs\htdocs_old\protected\controllers\TestController.php(10): CWizardController->init()
#11 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(275): TestController->init()
#12 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#13 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#15 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/
---
2013/09/24 09:22:27 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/htdocs_old/test/ranking
HTTP_REFERER=http://localhost/htdocs_old/test/
---
2013/09/24 09:22:30 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 20 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/24 09:22:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'wizard_log_id' in 'field list'. The SQL statement executed was: SELECT `t`.`yii_wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii_wizard_log where wizard_session_id = 20 group by action_id ) recent  on recent.wizard_log_id = t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\htdocs_old\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\htdocs_old\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(123): CActiveRecord->find()
#4 C:\xampp\htdocs\htdocs_old\protected\extensions\wizard\components\CWizardController.php(56): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#8 C:\xampp\htdocs\htdocs_old\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#9 C:\xampp\htdocs\htdocs_old\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\htdocs_old\index.php(48): CApplication->run()
#11 {main}
REQUEST_URI=/htdocs_old/test/ranking
---
2013/09/26 10:30:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'enquete.nl'' in C:\xampp\htdocs\paint\protected\lib\yii\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\paint\protected\lib\yii\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\paint\protected\lib\yii\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\paint\protected\lib\yii\base\CModule.php(388): CDbConnection->init()
#3 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\xampp\htdocs\paint\protected\lib\yii\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\xampp\htdocs\paint\protected\lib\yii\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\paint\protected\lib\yii\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(CWizardSession))
#7 C:\xampp\htdocs\paint\protected\extensions\wizard\models\CWizardSession.php(23): CActiveRecord::model('CWizardSession')
#8 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(75): CWizardSession::model()
#9 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(20): CWizardController->getSession()
#10 C:\xampp\htdocs\paint\protected\controllers\TestController.php(10): CWizardController->init()
#11 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(275): TestController->init()
#12 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('')
#13 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\paint\index.php(48): CApplication->run()
#15 {main}
REQUEST_URI=/paint/
---
2013/09/26 10:31:11 [error] [exception.CException] exception 'CException' with message 'CWizardModelAction and its behaviors do not have a method or closure named "getNext".' in C:\xampp\htdocs\paint\protected\lib\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\paint\protected\extensions\wizard\models\CWizardLog.php(160): CComponent->__call('getNext', Array)
#1 C:\xampp\htdocs\paint\protected\extensions\wizard\models\CWizardLog.php(160): CWizardModelAction->getNext(Object(CMap))
#2 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(46): CWizardLog->add(Object(CWizardModelAction))
#3 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#4 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#5 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#6 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#7 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\paint\index.php(48): CApplication->run()
#9 {main}
REQUEST_URI=/paint/test/ranking
HTTP_REFERER=http://localhost/paint/test/ranking
---
2013/09/26 10:33:10 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[t].[action_id] = 'answers' ORDER BY [t].wizard_log_id asc LIMIT 1' at line 1. The SQL statement executed was: SELECT `t`.`wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii.wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id=t.wizard_log_id WHERE [t].[action_id] = :action_id ORDER BY [t].wizard_log_id asc LIMIT 1.
2013/09/26 10:33:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[t].[action_id] = 'answers' ORDER BY [t].wizard_log_id asc LIMIT 1' at line 1. The SQL statement executed was: SELECT `t`.`wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii.wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id=t.wizard_log_id WHERE [t].[action_id] = :action_id ORDER BY [t].wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\paint\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\paint\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\paint\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\paint\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(49): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#8 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#9 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\paint\index.php(48): CApplication->run()
#11 {main}
REQUEST_URI=/paint/test/ranking
HTTP_REFERER=http://localhost/paint/test/ranking
---
2013/09/26 10:33:37 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.wizard_log' doesn't exist. The SQL statement executed was: SELECT `t`.`wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii.wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id=t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1.
2013/09/26 10:33:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.wizard_log' doesn't exist. The SQL statement executed was: SELECT `t`.`wizard_log_id`, `t`.`wizard_session_id`, `t`.`action_id`, `t`.`next_action_id`, `t`.`data_encoded`, `t`.`create_date`, `t`.`modify_date` FROM `yii_wizard_log` `t`  inner join (  select action_id, max(wizard_log_id) as wizard_log_id  from yii.wizard_log where wizard_session_id = :session_id  group by action_id ) recent  on recent.wizard_log_id=t.wizard_log_id WHERE t.action_id = :action_id ORDER BY t.wizard_log_id asc LIMIT 1' in C:\xampp\htdocs\paint\protected\lib\yii\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\paint\protected\lib\yii\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\paint\protected\components\ActiveRecord.php(115): CDbCommand->queryRow()
#2 C:\xampp\htdocs\paint\protected\lib\yii\db\ar\CActiveRecord.php(1392): ActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(122): CActiveRecord->find()
#4 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(49): CWizardController->retrieveState(Object(CWizardModelAction))
#5 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#6 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#7 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#8 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#9 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\paint\index.php(48): CApplication->run()
#11 {main}
REQUEST_URI=/paint/test/ranking
HTTP_REFERER=http://localhost/paint/test/ranking
---
2013/09/26 10:33:58 [error] [exception.CException] exception 'CException' with message 'TestController cannot find the requested view "_answer".' in C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php:879
Stack trace:
#0 C:\xampp\htdocs\paint\protected\components\CommonController.php(29): CController->renderPartial('_answer', Array, true, false)
#1 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(208): CommonController->renderPartial('_answer', Array, true, false)
#2 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(783): CWizardController->renderPartial('_answer', Array, true)
#3 C:\xampp\htdocs\paint\protected\components\CommonController.php(38): CController->render('_answer', Array, false)
#4 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(203): CommonController->render('_answer', Array, false)
#5 C:\xampp\htdocs\paint\protected\extensions\wizard\actions\CWizardModelAction.php(72): CWizardController->render('_answer', Array)
#6 C:\xampp\htdocs\paint\protected\extensions\wizard\components\CWizardController.php(63): CWizardModelAction->run()
#7 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(287): CWizardController->runAction(Object(CWizardModelAction))
#8 C:\xampp\htdocs\paint\protected\lib\yii\web\CController.php(266): CController->runActionWithFilters(Object(CWizardModelAction), Array)
#9 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(276): CController->run('ranking')
#10 C:\xampp\htdocs\paint\protected\lib\yii\web\CWebApplication.php(135): CWebApplication->runController('test/ranking')
#11 C:\xampp\htdocs\paint\protected\lib\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\paint\index.php(48): CApplication->run()
#13 {main}
REQUEST_URI=/paint/test/ranking
HTTP_REFERER=http://localhost/paint/test/ranking
---
